### PR TITLE
page layout

### PR DESCRIPTION
**Pull request by mim24sd**
In this pull request I want to:
1. add all pages in one page

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
 
 const App = () => {
   return (
     <BrowserRouter>
       <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/singles" element={<Singles />} />
-        <Route path="/albums" element={<Albums />} />
-        <Route path="/playlists" element={<PlayLists />} />
+        <Route path="/" element={<Home title="New albums and singles" />} />
```
Comment (blacksrc) [inline src/App.jsx]: - Please use children prop.
- Change the name to `MainLayout`.

Something like:
```jsx
<MainLayout title="Singles">
  <Singles />
</MainLayout>
```

File: src/pages/Home/index.jsx [hunk 2]
```diff
@@ -1,15 +1,25 @@
...
-const Home = () => {
+const Home = ({ content, title }) => {
+  let contentElement = <MusicList />;
+  if (content === "album") {
+    contentElement = <MusicList type={content} />;
+  } else if (content === "single") {
+    contentElement = <MusicList type={content} />;
+  } else if (content === "playlists") {
+    contentElement = <PlayLists />;
+  }
```
Comment (blacksrc) [inline src/pages/Home/index.jsx]: If you do [this](https://github.com/mim24sd/music-app/pull/8/files#r1058255975), then you can remove all of these checks.

File: src/App.jsx (hunk 1 above)
Comment (mim24sd) [reply src/App.jsx]: oooooh such a great idea.
I did it.
Thanks

File: src/pages/Home/index.jsx (hunk 2 above)
Comment (mim24sd) [reply src/pages/Home/index.jsx]: Yeees done

**Comment by blacksrc** [comment]
✅ LGTM

**Pull request merged**